pkg/dao/dinosaur.go

```diff
@@ -7,8 +7,28 @@ import (
 
 	"github.com/openshift-online/rh-trex/pkg/api"
 	"github.com/openshift-online/rh-trex/pkg/db"
+	"github.com/openshift-online/rh-trex/pkg/util"
 )
 
+var (
+	dinosaurTableName = util.ToSnakeCase(api.DinosaurTypeName) + "s"
+	dinosaurColumns   = []string{
```

Comment:
This means to support searching a new column, it must be added to this list?

I see [clusters-service implementing this requirement in this way](https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/blob/master/pkg/dao/clusters_const.go?ref_type=heads#L17), but AMS/OSDFM seem to [handle this differently](https://gitlab.cee.redhat.com/service/uhc-account-manager/-/blob/master/pkg/dao/account.go?ref_type=heads#L117).

I'm not sure which is better tbh, the AMS implementation is explicit while this implementation is simpler. @markturansky can you provide your input here?

@markturansky probably need your input here.

Reply:
Yeah, I'd like to eventually be able to move away from this to have something that uses reflect or something similar to figure it out :(

Reply:
i'd prefer to not explicitly set casing, as that negates the casing config allowed by gorm (that also does things like table name prefix, etc).  can we do this through gorm  somehow,?